pro_tes/ga4gh/tes/task_runs.py

```diff
@@ -213,6 +209,11 @@ def create_task(  # pylint: disable=too-many-statements,too-many-branches
             )
             return {"id": db_document.task.id}
 
+        # set TES state to SYSTEM_ERROR as task submission failed on all the available TES instances.
```

Comment:
Would be good to add another log message here indicating something like: `No suitable TES instance found. Task state set to 'SYSTEM_ERROR'.`

Reply:
Okay. I have replaced my comment with `No suitable TES instance found. Task state set to 'SYSTEM_ERROR'.` in the next commit.